tests/test_oic.py

```diff
@@ -143,6 +144,41 @@ def test_access_token_request(self):
         assert _eq(resp.keys(),
                    ['token_type', 'state', 'access_token', 'scope'])
 
+    def test_access_token_request_with_custom_response_class(self):
```

Comment:
This test should probably go to `test_oauth2_consumer` since the fix is there.

Reply:
Ok, I placed it at the `oic` level since `oauth2` is the underliying infrastructure so everything get tested altogether.  If you don't mind, I'm going to leave the test in `oic` and make another one in `test_oauth2_consumer`.

Reply:
The reasoning behind my comment was that the test should be where the change was.

But it sort of makes sense to test both, since we want to make sure that the oic doesn't overload completely.